src/commands/api-mesh/__tests__/log-get-bulk.test.js

```diff
@@ -291,3 +295,249 @@ describe('GetBulkLogCommand startTime and endTime validation', () => {
 		},
 	);
 });
+
+describe('GetBulkLogCommand with --past and --from flags', () => {
+	let parseSpy;
+
+	beforeEach(() => {
+		parseSpy = jest.spyOn(GetBulkLogCommand.prototype, 'parse').mockResolvedValue({
+			flags: {
+				past: '20mins',
+				from: '2025-03-09:12:00:00',
```

Comment:
Would this test fail if we go past 30 days from this date because of that 30day restriction we impose?